wikichron/dash/apps/networks/networks/models/CoEditingNetwork.py

```diff
@@ -34,9 +34,38 @@ class CoEditingNetwork(BaseNetwork):
     DIRECTED = False
 
     NETWORK_STATS = BaseNetwork.NETWORK_STATS.copy()
+    NETWORK_STATS['Gini of article ed.'] = 'gini_article_edits'
```

Comment:
These shorted metric names aren't clear to me: "Gini of article ed.", "Gini of close." "Gini of betwee.", etcetera.

I understand that you don't want to have too long names there, but I have checked it and there's enough space so they don't overflow the given width to every row on the network stats table. However, if you want to keep it like this, then I'd suggest to add an on :hover tooltip text for every metric so a longer description appears when the user mouse over the metric names.